<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmminorstrata</class>
 <widget class="QWidget" name="frmminorstrata">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lbHeader">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(247, 247, 247)</string>
     </property>
     <property name="text">
      <string>WRITE SOMETHING HERE LATER &lt;---</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Choose Minor Strata</string>
     </property>
     <property name="buddy">
      <cstring>tableView</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QTableView" name="tableView">
     <property name="toolTip">
      <string>Choose minor strata (or create new one)</string>
     </property>
     <property name="statusTip">
      <string>Choose minor strata (or create new one)</string>
     </property>
     <property name="whatsThis">
      <string>In this widget we can choose a minor strata, or create a new one</string>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Create New</string>
     </property>
     <property name="buddy">
      <cstring>lineNew</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineNew"/>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QGroupBox" name="groupDetails">
     <property name="title">
      <string>Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Group of Landing Sites</string>
          </property>
          <property name="buddy">
           <cstring>cmbGLS</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbGLS"/>
        </item>
       </layout>
      </item>
      <item row="0" column="3" rowspan="2" colspan="2">
       <widget class="QGroupBox" name="groupActivity">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Activity Status</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioActive">
           <property name="text">
            <string>Active</string>
           </property>
           <attribute name="buttonGroup">
            <string>buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="radioInactive">
             <property name="text">
              <string>Inactive</string>
             </property>
             <attribute name="buttonGroup">
              <string>buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmbReasons"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Comments</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPlainTextEdit" name="textComments">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Start Date</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>End Date</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" rowspan="2" colspan="2">
       <widget class="CustomTimeCtrl" name="customDtStart"/>
      </item>
      <item row="3" column="2" rowspan="2" colspan="3">
       <widget class="CustomTimeCtrl" name="customDtEnd"/>
      </item>
      <item row="4" column="4">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>488</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBackNext">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pushPrevious">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Previous</string>
          </property>
          <property name="icon">
           <iconset resource="app_new.qrc">
            <normaloff>:/app_new/back.png</normaloff>:/app_new/back.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushNext">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
          <property name="icon">
           <iconset resource="app_new.qrc">
            <normaloff>:/app_new/forward.png</normaloff>:/app_new/forward.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushNew">
     <property name="toolTip">
      <string>Create new record</string>
     </property>
     <property name="statusTip">
      <string>Create new record</string>
     </property>
     <property name="whatsThis">
      <string>Creates new record</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="app_new.qrc">
       <normaloff>:/app_new/filenew.png</normaloff>:/app_new/filenew.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lbSource">
     <property name="styleSheet">
      <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(220, 220, 220, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
font: 75 10pt &quot;Fixedsys&quot;;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>[SOURCE]</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTimeCtrl</class>
   <extends>QWidget</extends>
   <header>customtimectrl.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="app_new.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushNew</sender>
   <signal>clicked()</signal>
   <receiver>frmminorstrata</receiver>
   <slot>createRecord()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>686</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>frmminorstrata</receiver>
   <slot>onButtonClick(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>527</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioActive</sender>
   <signal>clicked(bool)</signal>
   <receiver>frmminorstrata</receiver>
   <slot>setActiveReason(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>frmminorstrata</receiver>
   <slot>previewRow(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioActive</sender>
   <signal>clicked(bool)</signal>
   <receiver>frmminorstrata</receiver>
   <slot>disableReasonCombo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioInactive</sender>
   <signal>clicked()</signal>
   <receiver>frmminorstrata</receiver>
   <slot>disableReasonCombo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showDetails()</slot>
  <slot>viewRecord()</slot>
  <slot>createRecord()</slot>
  <slot>onButtonClick(QAbstractButton*)</slot>
  <slot>setActiveReason(bool)</slot>
  <slot>previewRow(QModelIndex)</slot>
  <slot>disableReasonCombo()</slot>
  <slot>next()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
